.travis.yml

```diff
@@ -18,5 +18,4 @@ script:
   - cd ../libs/numpy/doc
   - make html
   - cd ../../../
-  - if [ "$PYTHON" == "/usr/bin/python" ]; then scons; fi
-
+  - if [ "$TRAVIS_PYTHON_VERSION" == "2.7" ]; then scons; fi
```

Comment:
Does this mean we don't run `scons` at all if python 3.5 is selected?

Reply:
I don't know why, but [it's just from the upstream](https://github.com/ndarray/Boost.NumPy/commit/4b14f7731c8cccd63a41adecd95642a302ccca3b).